frontend/src/helpers/dropdownMappers.ts

```diff
@@ -94,4 +95,8 @@ export function sortFromViasEntities(entities: Option[]){
     // Origins comes last
     return groupA === 'origins' ? 1 : -1;
   });
+}
+
+export function getViasGroupLabel(currentIndex: number | null) {
```

Comment:
If it can return the label for the origins I would give this method a different name: getGroupLabel only instead of getViasGroupLabel maybe?